### PR TITLE
-profile test

### PR DESCRIPTION
**Pull request by ewels**
Pipeline now runs using test data with nothing but:

```
nextflow run nf-core/methylseq -profile test -with-docker
```

Test data held on the `methylseq` branch of the nf-core/test-datasets repo: https://github.com/ewels/nf-core-test-datasets/tree/methylseq

### CODE REVIEW

**Comment by ewels** [comment]
Oops, accidentally scooped in a version number bump and a new singularity build script sorry. Ah well - those are both ready for the `dev` branch anyway..

File: Singularity [hunk 1]
```diff
@@ -0,0 +1,18 @@
+From:nfcore/base
+Bootstrap:docker
+
+%labels
+    MAINTAINER Phil Ewels <phil.ewels@scilifelab.se>
+    DESCRIPTION Docker image containing all requirements for the nf-core/methylseq pipeline
```
Comment (MaxUlysse) [inline Singularity]: Isn't it a Singularity image?

Comment (ewels) [reply Singularity]: hah, yes - copy & paste fail! ;)

**Comment by ewels** [comment]
Ok, tests now running instead of timing out! @pditommaso - you were right, it was the docker pull that was a problem. I moved this to the `before_install` part of the travis script instead and now they run fine.

Tests now failing because of a genuine problem with the pipeline... 🎉

**Comment by pditommaso** [comment]
Still not able to run as 

```
$ nextflow run nf-core/methylseq -profile test
N E X T F L O W  ~  version 0.30.0
Launching `nf-core/methylseq` [cheesy_koch] - revision: e9106e500a [master]
ERROR ~ No reference genome index specified!
```

Is that expected to work ?

**Comment by sven1103** [comment]
not on `master` branch yet, use `-r dev` please. The next patch version of `methylseq` should work then with your command.

**Pull request merged**